__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -12,6 +12,9 @@ const {
 } = e2eUtils;
 import { NESTED_COLUMNS_3_LEVELS } from './test-editor-data';
 
+const ANDROID_COLUMN_APPENDER_BUTTON_XPATH =
```

Comment:
I haven't checked it further, but seems the native composition for `Button` has changed in the new version of RN or dependencies. I experienced something similar on Gutenberg with other elements ([example](https://github.com/WordPress/gutenberg/pull/51303/commits/de279689b862b383af1eaf36e9282327ae31ec80)).